docs/api.md

```diff
@@ -61,7 +61,14 @@ An optional function that will be called each time a `PATCH` has been successful
 
 _Default value:_ `null`
 
-An optional function called when the upload finished successfully.
+An optional function called when the upload finished successfully. The argument will be an object with information about the event. Its `lastResponse` key will contain the lastly received HttpResponse, which can be used to retrieve additional data from the server. For example:
```

Comment:
```suggestion
An optional function called when the upload finished successfully.

The argument will be an object with information about the completed upload. Its `lastResponse` property contains lastly received [`HttpResponse`](#httpstack), which can be used to retrieve additional data from the server. Be aware that this is usually a response to a `PATCH` request, but it might also be a response for a `POST` request (if `uploadDataDuringCreation` is enabled or an empty file is uploaded) or `HEAD` request (if an already completed upload is resumed).
```

Some more details about the response.